django/db/__init__.py

```diff
@@ -35,6 +35,8 @@
     "DJANGO_VERSION_PICKLE_KEY",
 ]
 
+ALL_DATABASES = "__all__"
```

Comment:
I think we can drop the constant, other uses of `__all__` to mean "all databases" don' use it.

Reply:
I was trying to be too fancy 😄